internal/tfimport/tfimport.go

```diff
@@ -538,19 +573,52 @@ func Run(rn runner.Runner, importRn runner.Runner, runArgs *RunArgs) error {
 		return fmt.Errorf("expand path %q: %v", inputDir, err)
 	}
 
+	var successesTotal, failuresTotal []string
 	skipped := make(map[string]bool)
+	var ie *importErr
```

Comment:
Can you just use ie.ErrMsgs instead of failuresTotal?

Reply:
No, since ie may be nil.